frog/imports/client/TeacherView/utils/exportComponent.js

```diff
@@ -86,7 +87,10 @@ export const generateExport = (
       .join('\n');
     const userfile = 'userid\tusername\tinstanceid\n' + mappings;
 
-    img.file('usermappings-' + (item.groupingKey || '???') + '.tsv', userfile);
+    folder.file(
+      'usermappings-' + (item.groupingKey || '???') + '.tsv',
+      userfile
+    );
```

Comment:
Renaming for the code to be clearer. There was no images involved so `img` was not a good name